### PR TITLE
MM-38093: updates notifications copy on CRT replies

### PR DESCRIPTION
**Pull request by koox00**
#### Summary

Desktop notifications title for replies when users have with CRT "on"
should show "Reply in channel name/Direct Message".

e.g:
- Reply in Off-Topic
- Reply in Direct Message

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-38093

#### Related Pull Requests

- Has server changes (https://github.com/mattermost/mattermost-server/pull/18332)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: actions/notification_actions.jsx [hunk 1]
```diff
@@ -115,6 +115,10 @@ export function sendDesktopNotification(post, msgProps) {
             }
         }
 
+        if (isCrtReply) {
+            title = Utils.localizeMessage('notification.crt', 'Reply in ') + title;
```
Comment (hmhealey) [inline actions/notification_actions.jsx:119]: Concatenating the title onto `Reply in` might not work depending on the language. We can use `Utils.localizeAndFormatMessage` to let translators order the title the way they want

**Review by hmhealey — CHANGES_REQUESTED** [review]

**Review by Willyfrog — APPROVED** [review]
other than @hmhealey suggestion, looks good

**Review by hmhealey — APPROVED** [review]

**Pull request merged**